### DISCUSSION TITLE
Compatibility raspberry pi 4 (ARM v7)

### DISCUSSION THREAD

**Discussion by flavio-cbz** [FlareSolverr/FlareSolverr#83] [Q&A]
Is FlareSolverr compatible with arm v7 if not intended?

**Comment by ngosang** [comment]
Yes, there are Docker images compatible with all Raspberry Pi versions (ARM v7)
Docker images are available in:
* GitHub Registry => https://github.com/orgs/FlareSolverr/packages/container/package/flaresolverr
* DockerHub => https://hub.docker.com/r/flaresolverr/flaresolverr

**Reply by flavio-cbz** [reply]
ty so much

**Reply by ErasGalban** [reply]
is there any version, where I don't have to install Docker?

**Reply by ngosang** [reply]
No sorry, but you can install from source code. Read the readme :)
